### ISSUE TITLE
ChatPage: Make messages more compact

### ISSUE THREAD

**Issue by LNJ2** [KaidanIM/Kaidan#158]
Currently the messages in the chat just use too much space ...
e.g. the date could be made much smaller

**Comment by IlyaBizyaev** [comment]
That's true; dates and times could also be displayed Telegram-way (i.e. date for a day, time out of bubble).

Message font size, however, needs to be slightly bigger.
![spectacle m30831](https://user-images.githubusercontent.com/11175213/34073985-131aa946-e2b7-11e7-99f8-d779ea80739e.png)
